cypress/integration/common/index.ts

```diff
@@ -0,0 +1,10 @@
+import { And } from "@badeball/cypress-cucumber-preprocessor/methods";
```

Comment:
would have been nice to at least put in an existing file, so not another file was yet introduced.
`wallet.e2e.ts` also contains "modelContent", or else, there is just `deals.e2e.ts`

Reply:
I added these steps in the `index.ts` file because they look rather general. A modal can be displayed on any page for any purpose. I thought we can use these in the future for some other tests.